corehq/apps/domain/templates/domain/admin/info_basic.html

```diff
@@ -6,6 +6,17 @@
 {% block js-inline %} {{ block.super }}
 <script type="text/javascript">
     $(function() {
+        // Update project dropdown in page header with potentially new project name
+        var hr_name = $('#id_hr_name').val(),
+            $projectsMenu = $('#nav-projects-menu'),
+            currentHref = $projectsMenu.find(".btn-project-active").attr("href");
+        _.each($projectsMenu.find("a"), function(link) {
+            var $link = $(link);
+            if (currentHref.startsWith($link.attr("href"))) {
```

Comment:
For the links in the dropdown, yes, they're equal. But the link on the actual button, to the current project, goes to that project's settings page, so it isn't quite the same.